Atomic/storage.py

```diff
@@ -20,6 +20,9 @@
     from atomic import Atomic
 
 class Storage(Atomic):
+    dss_conf = "/etc/sysconfig/docker-storage-setup"
+    dss_conf_bak = "/etc/sysconfig/docker-storage-setup" + ".bkp"
```

Comment:
This is better written as `dss_conf_bak =  dss_conf + ".bkp"`.  (This was in my original code, sorry.)